.idea/modules/alpha_main.iml

```diff
@@ -21,6 +21,7 @@
     <orderEntry type="library" name="Gradle: org.apache.commons:commons-collections4:4.1" level="project" />
     <orderEntry type="library" name="Gradle: org.apache.commons:commons-lang3:3.4" level="project" />
     <orderEntry type="library" name="Gradle: ch.qos.logback:logback-classic:1.1.7" level="project" />
+    <orderEntry type="library" name="Gradle: com.google.guava:guava:20.0" level="project" />
```

Comment:
Google Guava is licensed under the Apache License 2.0, which is (conceptually) derived from MIT and BSD. There's some discussion about it on [SE](http://softwareengineering.stackexchange.com/questions/40561).

The APL-2.0 says:

> For the purposes of this License, Derivative Works shall not include works that remain separable from, or merely link (or bind by name) to the interfaces of, the Work and Derivative Works thereof.

Alpha remains separable from APL, as we can (theoretically) just use different implementations of their interfaces.

With this definition I would say that Alpha is not a Derivative Work of Google Guava, so we don't have to do anything. If you for some reason would see Alpha as a Derivative Work, we would have to include the original license (APL-2.0) and copyright of Google Guava. Google Guava does not include a NOTICE file, so nothing to do there.

By the way: We also depend on ANTLR (Licensed under BSD) and Apache Commons (licensed under APL-2.0) in a similar way.